### ISSUE TITLE
Add relator codes for places on institution records

### ISSUE THREAD

**Issue by ahankinson** [rism-digital/muscat#1088]
The 551 field (751 if looking to make this change after #1087 ) does not allow $i (relationship information) on institutions, but it does for people. This should be changed to allow $i on 551/751.

A separate issue addresses what values should actually be added for these relationships.

**Comment by xhero** [comment]
@jenniferward is it OK to add a $i relator code here? Do we have a list we can use?

**Comment by jenniferward** [comment]
I need to double-check with our Series C colleagues - though this isn't one of "their" fields, they might have an interest in this.

**Comment by Docudoctor** [comment]
Hi,
551 is related to 110 $c. The places in 551 are standardized and can differ from 110 $c
Example: Institution ID 51005815
Triton was an historical institution in Leningrad. Today's name of place is Sankt-Peterburg. 
Any relator codes needed for 551 due to that?

Best,
Guido

**Comment by jenniferward** [comment]
I think we should add it, though it probably won't have the variety that we have in People.
What would we use in Guido's example?  https://muscat.rism.info/admin/institutions/51005815
